src/components/index-page/css/index-scorecard.css

```diff
@@ -0,0 +1,34 @@
+.index-scorecard {
+  @apply parent-section;
+  @apply bg-launch-neutral-10;
+}
+
+.index-scorecard__contents {
+  @apply contents-container;
+  @apply flex-col md:flex-row md:pt-4 md:space-x-6;
+}
+
+.index-scorecard__image-container {
+  @apply py-8 lg:p-0 w-full lg:w-5/12 text-center;
+}
+
+.index-scorecard__image {
+  @apply rounded-image bg-launch-black;
+}
+
+.index-scorecard__text-container {
+  @apply flex flex-col md:items-start lg:pr-16;
+  @apply text-left w-full lg:w-7/12;
```

Comment:
fyi I probably would use a grid for this with a 12 column configuration - it basically does the same thing but gives us a nice tailwind abstraction for it.